### PR TITLE
tests/crypto_aesctr: check nonce

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

File: tests/crypto_aesctr/main.c [hunk 1]
```diff
@@ -410,6 +474,17 @@ selftest(void)
 	if (selftest_cases(tests_256, num_tests, 0))
 		failures++;
 
+	/* Test with nonce=*2 (decimal). */
```
Comment (cperciva) [inline tests/crypto_aesctr/main.c:477]: I'm confused by "nonce=*2 (decimal)" here.  It looks like you've set a nonce of "2222222" in base 1024?

Comment (cperciva) [reply tests/crypto_aesctr/main.c:477]: I mean, there isn't anything especially wrong with this value, but it's a strange choice.  I would have gone with something 0x1234567890ABCDEF.

Comment (gperciva) [reply tests/crypto_aesctr/main.c:477]: I wanted something higher than `0x8` to check that it wasn't `int64_t` or something like that.  How about `0xfedcba987654321`?

Comment (cperciva) [reply tests/crypto_aesctr/main.c:477]: I think you mean 0xfedcba9876543210, not 0xfedcba987654321, but sure.

I think the nonce value you were calculating was 0x4010040100401004, so not high enough to run into signed issues anyway...

**Comment by gperciva** [comment]
Updated with nonce = `0xfedcba9876543210`, ready for merge.

**Pull request merged**